AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/MAX/Rewarded/ALMAXRewardedAdViewController.m

```diff
@@ -102,7 +102,7 @@ - (void)didCompleteRewardedVideoForAd:(MAAd *)ad
     [self logCallback: __PRETTY_FUNCTION__];
 }
 
-- (void)didRewardUserForAd:(MAAd *)ad withReward:(MAReward *)reward
+- (void)didPayRevenueForAd:(MAAd *)ad withReward:(MAReward *)reward
```

Comment:
Not sure if this is a valid callback.

Reply:
I'd definitely cmd + B the test apps to see if there are warnings next time when submitting PRs.